examples/vae/utils.py

```diff
@@ -0,0 +1,90 @@
+"""
+    This code is created with reference to torchvision/utils.py.
+
+    Modify: torch.tensor -> jax.numpy.DeviceArray
+
+    If you want to know about this file in detail, please visit the original code:
+        https://github.com/pytorch/vision/blob/master/torchvision/utils.py
+"""
+import math
+import jax
+import jax.numpy as np
+
+irange = range
```

Comment:
We try to not shadow globally defined names so I would prefer to change the argument named range to something else.

Reply:
Fix it.